README.md

```diff
@@ -8,6 +8,53 @@ J. H. Legarreta, M. Descoteaux, and P.-M. Jodoin. “PROCESSING OF TRACTOGRAPHY
 RESULTS USING AN AUTOENCODER”. Filed 03 2021. Imeka Solutions Inc. United States
 Patent #17/337,413. Pending.
 
+## Installation
+
+To use tractolearn, it is recommended to create a virtual environment using python 3.8 that
+will host the necessary dependencies:
+
+```sh
+   virtualenv tractolearn_env --python=python3.8
+   source tractolearn_env/bin/activate
+```
+
+They can be installed by executing:
+
+```sh
+   pip install -r requirements.txt
+   pip install -r requirements_dmri.txt
+```
+
+Torch not included. Tested with an NVIDIA RTX 3090 with:
```

Comment:
Is there a reason for this? They should be in the `setup.cfg` version. Anyways I will specify them when transitioning to PEP0518.

Reply:
Added in the requirements.txt

Reply:
Done